www-plugins/passff-host/passff-host-1.0.1.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_4 python3_5 python3_6 )
+
+inherit python-single-r1
+
+DESCRIPTION="Host app for the PassFF WebExtension"
+HOMEPAGE="https://github.com/passff/passff-host"
+
+# Difference between git and release
+# See https://github.com/passff/passff-host/issues/18
+SRC_URI="
+	https://github.com/passff/passff-host/raw/${PV}/src/passff.py -> ${P}.py
+	https://github.com/passff/passff-host/raw/${PV}/src/passff.json -> ${P}.json
+"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="chrome chromium firefox vivaldi"
```

Comment:
Unless I'm missing something, `USE=-*` will turn this into a package installing no files. I don't think that is really desirable, so you probably want to do `REQUIRED_USE` with `||` to require at least one flag enabled.